doc/architecture/decisions/0012-use-aws-for-hosting.md

```diff
@@ -0,0 +1,17 @@
+# 1. use-aws-for-hosting
+
+Date: 2022-04-07
+
+## Status
+
+Accepted
+
+## Context
+
+This ADR is to document a past decision retrospectively for reference. The context for this decision is now unknown.
+
+## Decision
+
+The hosting of the live service will exist in an AWS account owned and maintained by the Warwick Business School.
+
+## Consequences
```

Comment:
Like above, without the context I can only guess at the consequences weighed up at the time. Anything I can include here?

Reply:
Could go with:

```
This will require manual deployment of the application each time code changes are made. 
There is a risk with personnel changes, that any new people may not have the same 
knowledge as the existing team, so may not be able to deploy the application. Also, any
security updates may not be deployed as rapidly without a CI pipeline. This is something
we will review as the project matures.
```